<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="DataNext Test Cases"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="2.846">
  <properties />
            <testcase name="liabilityFlow_CreateLeadTest" time="2.846" classname="com.crm.testcases.LiabilityFlow_CreateLeadTest">
                <failure type="java.lang.NullPointerException"
                  message="null value in entry: url=null"
              >
<![CDATA[java.lang.NullPointerException: null value in entry: url=null
  com.google.common.collect.CollectPreconditions.checkEntryNotNull(CollectPreconditions.java:33)
  com.google.common.collect.SingletonImmutableBiMap.<init>(SingletonImmutableBiMap.java:43)
  com.google.common.collect.ImmutableBiMap.of(ImmutableBiMap.java:81)
  com.google.common.collect.ImmutableMap.of(ImmutableMap.java:126)
  org.openqa.selenium.remote.DriverCommand.GET(DriverCommand.java:191)
  org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:304)
  com.crm.base.TestBase.setUp(TestBase.java:136)
  com.crm.testcases.LiabilityFlow_CreateLeadTest.liabilityFlow_CreateLeadTest(LiabilityFlow_CreateLeadTest.java:24)
  java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
  java.base/java.lang.reflect.Method.invoke(Method.java:577)
  org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
  org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
  org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
  org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
  org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
  org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
  org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
  org.testng.TestRunner.privateRun(TestRunner.java:808)
  org.testng.TestRunner.run(TestRunner.java:603)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
  org.testng.SuiteRunner.run(SuiteRunner.java:326)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.runSuites(TestNG.java:1092)
  org.testng.TestNG.run(TestNG.java:1060)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
